backend/danswer/server/documents/cc_pair.py

```diff
@@ -33,6 +39,38 @@
 router = APIRouter(prefix="/manage")
 
 
```

Comment:
I created a seperate endpoint to get the index attempts for a couple reasons:

- enable pagination of the index attempts
- allow the front end to load the rest of the connector data without having to wait for the index attempts